### DISCUSSION TITLE
MacBookPro18,1/18,2 stopped controlling fans after upgrade to macOS Sonoma 14.4 beta

### DISCUSSION THREAD

**Discussion by arkdev1** [crystalidea/macs-fan-control#732] [Q&A]
Good morning everyone, for a few days the program no longer controls the fans as before, I can identify this problem since I have installed Sonoma Beta Version of 14.4 (23E5180j)

Some idea how to solve?

**Comment by kleuter** [comment]
Same for MacBookPro18.2
![image](https://github.com/crystalidea/macs-fan-control/assets/2600624/744a815f-3d15-4085-8c70-f30570aa1e54)
![image](https://github.com/crystalidea/macs-fan-control/assets/2600624/8028089d-ab68-4221-9caa-ad5454510695)

**Comment by kleuter** [comment]
Can you try to do some CPU intensive task so the system will turn on the fans and then try to set some custom RPM value (e.g. Full Blast)?

**Reply by arkdev1** [reply]
Hi @kleuter, trying now:

![image](https://github.com/crystalidea/macs-fan-control/assets/138211005/a43e83e8-f096-4b8d-988e-a32a49ea759b)

I'm a little worried 60°C and no fans?
Could it be some Sonoma problem?

**Comment by kleuter** [comment]
https://github.com/crystalidea/macs-fan-control/issues/733

**Reply by kleuter** [reply]
That's how their firmware works: try 70C

**Reply by arkdev1** [reply]
No sign from the fans, plus I didn't hear any rustling or similar when I tried.
Above 70°C seems like an exaggeration to me.
Not an expert, just a regular user of the plugin

In the screen is 70.8, but i went over 72.5

![Screenshot 2024-02-02 alle 18 53 05](https://github.com/crystalidea/macs-fan-control/assets/138211005/d5e60b77-e1cb-4592-bd50-f368fd726735)
